File: app/src/main/java/org/neidhardt/dynamicsoundboard/notifications/NotificationIds.java
package org.neidhardt.dynamicsoundboard.notifications;

/**
 * Created by eric.neidhardt on 15.12.2014.
 */
public class NotificationIds
{
	public static final int NOTIFICATION_ID_PLAYLIST = "org.neidhardt.dynamicsoundboard.notifications.NOTIFICATION_ID_PLAYLIST".hashCode(); // must be a random, unique id

	public static final String KEY_PLAYER_ID = "org.neidhardt.dynamicsoundboard.notification.SoundPlayingNotification.KEY_PLAYER_ID";
	public static final String KEY_NOTIFICATION_ID = "org.neidhardt.dynamicsoundboard.notification.SoundPlayingNotification.KEY_NOTIFICATION_ID";

	public static final String ACTION_DISMISS = "org.neidhardt.dynamicsoundboard.notification.SoundPlayingNotification.ACTION_DISMISS";
	public static final String ACTION_PLAY = "org.neidhardt.dynamicsoundboard.notification.SoundPlayingNotification.ACTION_PLAY";
	public static final String ACTION_PAUSE = "org.neidhardt.dynamicsoundboard.notification.SoundPlayingNotification.ACTION_PAUSE";
	public static final String ACTION_STOP = "org.neidhardt.dynamicsoundboard.notification.SoundPlayingNotification.ACTION_STOP";
	public static final String ACTION_FADE_OUT = "org.neidhardt.dynamicsoundboard.notification.SoundPlayingNotification.ACTION_FADE_OUT";
}

File: app/src/main/kotlin/org/neidhardt/dynamicsoundboard/notifications/NotificationIds.kt
package org.neidhardt.dynamicsoundboard.notifications;

import java.util.*

/**
 * File created by eric.neidhardt on 15.12.2014.
 */
object NotificationConstants {
	private val appUUID = UUID.randomUUID().toString()

	val NOTIFICATION_ID_PLAYLIST: Int = (appUUID + "NOTIFICATION_ID_PLAYLIST").hashCode()

	const val KEY_PLAYER_ID: String = "KEY_PLAYER_ID"
	const val KEY_NOTIFICATION_ID: String = "KEY_NOTIFICATION_ID"

	val ACTION_DISMISS: String = appUUID + "ACTION_DISMISS"
	val ACTION_PLAY: String = appUUID + "ACTION_PLAY"
	val ACTION_STOP: String = appUUID + "ACTION_STOP"
	val ACTION_FADE_OUT: String = appUUID + "org.neidhardt.dynamicsoundboard.notification.SoundPlayingNotification.ACTION_FADE_OUT"
}

